Documentation/ceph-upgrade.md

```diff
@@ -57,6 +57,11 @@ those releases.
 * Rook no longer deploys Prometheus rules from the operator. If you have been relying on Rook to
   deploy prometheus rules in the past, please follow the upgrade instructions [below](#prometheus).
 
+* Due to a number of Ceph issues and changes, Rook officially only supports Ceph
+  v16.2.7 or higher for CephNFS. If you are using an earlier version, we highly recommend you
+  upgrade your Ceph version following the advice given in Rook's
+  [v1.8 NFS docs](https://rook.github.io/docs/rook/v1.8/ceph-nfs-crd.html).
```

Comment:
Of note, we don't stop anyone from using v16.2.6 or below, but with this note, we do note that we officially drop support for those Ceph versions along with NFS. Do you think the message is clear? Does the approach seem good?

Reply:
Makes sense, just a small suggestion on the wording added separately